src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -151,6 +151,13 @@ A placeholder shows for lazy-loaded iframes that are still being fetched.
 
 ## FAQ
 
+### Are there plans to expand this feature?
+
+There are plans to change the default lazy-loading behavior of the browser to automatically
```

Comment:
nit: Would it be possible to mention here that Chrome would only target images and iframes that it determines to be good candidates for lazy loading? I'm worried that developers might be worried that all images and iframes will be lazily loaded indiscriminately for Lite Mode users

Reply:
That's a great point. Updated in #1280